bravado/mapping/docstring.py

```diff
@@ -58,3 +58,65 @@ def create_model_docstring(props):
             docstring += ": " + props[prop]['description']
         docstring += '\n\t'
     return docstring
+
+
+def create_operation_docstring(json_):
+    """Builds Operation docstring from the json dict
+
+    :param json_: data to create docstring from
+    :type json_: dict
```

Comment:
I started calling the "fragments" from the spec as <name>_dict but I think <name>_spec is more appropriate too. I'll try to make the change globally in the next pull request.